README

```diff
@@ -90,7 +90,7 @@ Let's rewrite the example using the Python DSL:
 .. code:: python
 
     from elasticsearch import Elasticsearch
```

Comment:
Shouldn't we import from elasticserch2 ? `from elasticsearch2 import Elasticsearch`